tests/new_unit_tests/Commands/Auth/WhoamiCommandTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Auth;
+
+use Consolidation\OutputFormatters\StructuredData\AssociativeList;
+use Pantheon\Terminus\Commands\Auth\WhoamiCommand;
+
+class WhoamiCommandTest extends AuthTest
+{
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->command = new WhoamiCommand();
+        $this->command->setConfig($this->config);
+        $this->command->setLogger($this->logger);
+        $this->command->setSession($this->session);
+    }
+
+    /**
+     * Exercises WhoamiCommand::WhoAmI When the user is logged in
+     */
+    public function testWhoAmI()
+    {
+        $email = "email@ddr.ess";
+        $user = $this->getMockBuilder(User::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $user->expects($this->once())
```

Comment:
My unit tests keep failing on fetch and serialize. Not sure why these statements aren't getting through.